mobile/apps/artist/templates/header.jade

```diff
@@ -1,11 +1,11 @@
--var hometown = artist.hometown || artist.get('hometown')
--var years = artist.years || artist.get('years')
--var name = artist.name || artist.get('name')
+-var hometown = artist.hometown || (typeof artist.get === 'function' ? artist.get('hometown') : null)
```

Comment:
This is sort of messy. But, if the artist data comes from metaphysics and the artist has no `hometown`, this will fail on a `cannot read property 'get' of undefined`. I'm not sure if there's a better way to guard against `hometown: null` case (or any of the other fields).